Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>app/data/db_credentials\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(revision 1732d405efc5ce23da4a516001d182fc375f34b9)
+++ .gitignore	(date 1601465000483)
@@ -1,1 +1,2 @@
 app/data/db_credentials
+.venv/
